.github/configurations/vllm-tensorizer.yml

```diff
@@ -3,4 +3,6 @@ vllm-commit:
 flashinfer-commit:
   - 'v0.2.6.post1'
 base-image:
-  - 'ghcr.io/coreweave/ml-containers/torch-extras:es-compute-12.0-67208ca-nccl-cuda12.9.0-ubuntu22.04-nccl2.27.3-1-torch2.7.1-vision0.22.1-audio2.7.1-abi1'
+  - 'ghcr.io/coreweave/ml-containers/torch-extras:es-cuda-12.9.1-74755e9-nccl-cuda12.9.1-ubuntu22.04-nccl2.27.5-1-torch2.7.1-vision0.22.1-audio2.7.1-abi1'
+lean-base-image:
+  - 'ghcr.io/coreweave/ml-containers/torch-extras:es-cuda-12.9.1-74755e9-base-cuda12.9.1-ubuntu22.04-torch2.7.1-vision0.22.1-audio2.7.1-abi1'
```

Comment:
Pointing out for the record here that this PR is also updating the base image to one with CUDA 12.9.1 up from CUDA 12.9.0